docs/modules/ROOT/pages/cop_api_v1_changelog.adoc

```diff
@@ -0,0 +1,258 @@
+= Cop API v0 to v1 API changes
```

Comment:
Probably we can name this upgrade notes and put here everything major for the update, not just the Cop API changes.

Reply:
Renamed it 👍